app/models/qualifications.rb

```diff
@@ -0,0 +1,27 @@
+class Qualifications
+  def initialize(profpost_flag:, is_pgde:, is_further_education:)
+    @profpost_flag = profpost_flag
+    @is_pgde = is_pgde
+    @is_further_education = is_further_education
+  end
+
+  def to_a
+    qts_or_not + uni_qualification_or_not
+  end
+
+private
+
+  def qts_or_not
+    @is_further_education ? [] : [:qts]
+  end
+
+  def uni_qualification_or_not
+    if @is_pgde
+      [:pgde]
+    elsif @profpost_flag == "recommendation_for_qts"
```

Comment:
Any chance to rid of magic string?

Reply:
I can't find any. I think the fact that it's being used in the comparison with the `profpost_flag` should link it to that.

Reply:
== `"recommendation_for_qts"` 

old habits, never comfortable seeing comparing to a hardcoded string

Reply:
much better but what it is awar`e`ded

Reply:
I've gotten rid of the `Qualifications` class and put everything into a concern, so this point is now fixed: 89e834d